src/SFA.DAS.AdminService.Web/Controllers/Roatp/Apply/RoatpGatewayOrganisationChecksController.cs

```diff
@@ -23,6 +23,7 @@ public class RoatpGatewayOrganisationChecksController : RoatpGatewayControllerBa
         private readonly ILogger<RoatpGatewayOrganisationChecksController> _logger;
         private readonly IGatewayOrganisationChecksOrchestrator _orchestrator;
 
+        private const string GatewayViewsLocation = "~/Views/Roatp/Apply/Gateway/pages/";
```

Comment:
Could it be done without the trailing slash. Looks a bit un-natural when viewing the paths to the view down below